### DISCUSSION TITLE
overriding `nullable` property

### DISCUSSION THREAD

**Discussion by danicc097** [swaggest/openapi-go#74] [General]
Is it possible to set `nullable: false` in struct fields with `required: true` tag? Using as tag isn't recognized by jsonschema-go and I cannot seem to override with `InterceptSchema` since it's not exposed. 
My use case is that slices and maps with `required: true` always generate `nullable: true` regardless.

**Comment by danicc097** [comment]
It looks like `InterceptProp` enables what I need, since field tag and schema are both accesible so I can implement custom `nullable` tag logic

**Reply by vearutop** [reply]
There is also `InterceptNullability` that might be a better fit than `InterceptProp`.
